### PR TITLE
making sure the flag opens the Unlock site in a new window

### PR DESCRIPTION
**Pull request by julien51**
This will fix the bug where clicking on the link opens inside the flag...
Not sure how to test this?

- [X] 1 PR, 1 purpose: my Pull Request applies to a single purpose
  - [ ] This PR only contains configuration changes (package.json, etc.)
  - [X] This PR only contains code changes (if configuration changes are required, do a separate PR first, then re-base)
- [X] My code follows the style guidelines of this project, including naming conventions
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If my code adds or changes components, I have written corresponding stories with Storybook
- [X] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

**Comment by julien51** [comment]
Thank you for reviewing so quickly ;)

**Comment by cellog** [comment]
we can assert on the dom element's attributes

```js
expect(getByText('Unlock').attributes.target).toBe('_blank')
```

or something like that

File: paywall/src/components/lock/UnlockFlag.js [hunk 1]
```diff
@@ -177,7 +179,7 @@ const Flag = styled(Colophon).attrs({
         grid-row: 2;
         align-self: start;
       }
-  
+
```
Comment (julien51) [inline paywall/src/components/lock/UnlockFlag.js]: prettier?

**Comment by julien51** [comment]
This will probably break the snapshots :/ Pushing a change.

**Comment by julien51** [comment]
Adding the test too.

**Pull request merged**